packages/ui/src/composites/wui-wallet-image/styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from 'lit'
+
+export default css`
+  div {
+    position: relative;
+    border-radius: 12px;
+    overflow: hidden;
+    background: var(--wui-overlay-002);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  div::after {
```

Comment:
When using stylistic pseudo elements like this that overlay original, it's good practice to set `pointer-events: none` just in case. There might be cases where it covers something up that is interactive. I'm not sure, but could be that in this case it prevents user from right clicking and saving an image or something?